src/org/myworldgis/netlogo/VectorFeature.java

```diff
@@ -252,6 +252,14 @@ public Envelope getEnvelope () {
         return _geometry.getEnvelopeInternal();
     }
 
+    /** */
+    public double getDistance (Geometry geometry) {
+        if( _geometry ===null|| geometry === null ||geometry.isEmpty()) {
```

Comment:
Java uses `==` to check equality, not `===`.  Also the spacing needs to be fixed with spaces around the `||` operators.  We also prefer a space after the keyword and before the open paren, so `if (_geometry...`